css/style.css

```diff
@@ -3,3 +3,4 @@
     #main-doc {margin-left: 216px;}
   }
 
```

Comment:
## Please remove this empty line it will make your code look better and might solve a linter error. 🙏